docs/kubernetes.rst

```diff
@@ -32,6 +32,55 @@ Configuration
 
 Patroni Kubernetes :ref:`settings <kubernetes_settings>` and :ref:`environment variables <kubernetes_environment>` are described in the general chapters of the documentation.
 
+Customize role label
+^^^^^^^^^^^^^^^^^
+By default, Patroni will set corresponding labels on the pod it runs in based on node's role, such as ``role=master``.
+The key and value of label can be customized by `kubernetes.role_label`, `kubernetes.leader_label_value`, `kubernetes.follower_label_value` and `kubernetes.standby_leader_label_value`.
+
+Note that if you migrate from default role labels to custom ones, you can reduce downtime by following migration steps:
+
+1. Add a temporary label using original role value for the pod with `kubernetes.tmp_role_label`(like ``tmp_role``). Updated pod looks like:
+
```

Comment:
I would better prefer using `master`/`primary` as an example. This is something most people will have to deal with, while `replica` probably stay as it is.
Or, showing two different cases (primary and replica) would be even better.

Reply:
> Updated pod looks like:

I would probably rephrase it as: `Once pods are restarted they will get following labels set by Patroni:`

Reply:
I think a master/primary would be good enough